src/test/robot/api/storage/data/read_events.js

```diff
@@ -0,0 +1,39 @@
+var sharedConfig = {
+    "storage": {
+        "namespace": SCOPE_NAME
+    }
+}
+
+PreconditionLoad
+	.config(sharedConfig)
+	.config({
+		"item" : {
+            "data" : {
+                "size" : "1000KB"
```

Comment:
the bug related to the 1MB events write was fixed in the current pravega version already, so this option may be skipped

Reply:
done